### ISSUE TITLE
Energy monitor stopped working

### ISSUE THREAD

**Issue by domramsey** [bigbadblunt/homeassistant-lightwave2#31]
At some point in the last couple of weeks the energy monitor component of this integration has stopped working correctly. It will work for a few minutes after restarting HA or even just updating the integration options, but then it stops updating.

Lightwave light switches are unaffected. There's no specific error in the logs. Energy readings are correctly updated within the official Lightwave app.

**Comment by bigbadblunt** [comment]
Hi, I can't reproduce. Can you update to the latest version (this has a couple of stability fixes in the underlying library).

If it persists, can you please attach a log file with debug-level logging.

**Comment by domramsey** [comment]
Thanks. Attached a log with the only output from the component. 

I've tried updating, completely removing and reinstalling, different versions. They're all doing the same for me now. The electricity monitor works for a few minutes after HA startup, then just stops updating.

[lightwave2_log.txt](https://github.com/bigbadblunt/homeassistant-lightwave2/files/6189717/lightwave2_log.txt)

**Comment by bigbadblunt** [comment]
Can you please add logging for the underlying library:
```
  logs:
    lightwave2.lightwave2: debug
    custom_components.lightwave2: debug
```

**Comment by domramsey** [comment]
Ok, attached. Apologies, it's quite large but I wanted to include all the startup plus where I noticed issues. If you look at 18:10 in the log file, the energy monitor stopped sending data.

At 18:40 I turned off the Lightwave light switch, which worked and also restarted the energy monitor sending data.

[lightwave_log.txt](https://github.com/bigbadblunt/homeassistant-lightwave2/files/6199800/lightwave_log.txt)
